cheddar/cheddar-integration-aws/src/test/java/com/clicktravel/cheddar/infrastructure/persistence/database/tx/TransactionalDatabaseTemplateTest.java

```diff
@@ -546,10 +544,7 @@ private StubItem randomStubItem() {
         final StubItem item = new StubItem();
         item.setId(randomId());
         item.setStringProperty(randomString(10));
-        item.setStringProperty2(randomString(10));
-        item.setBooleanProperty(randomBoolean());
         item.setVersion(randomLong());
-        item.setStringSetProperty(Sets.newSet(randomString(10), randomString(10), randomString(10)));
```

Comment:
Using a simpler Item bean to illustrate what is happening. It only has one property: stringProperty.